tools/init.sh

```diff
@@ -59,7 +59,13 @@ case "$1" in
            done
 
            PEER=$($TOPOS_BIN tce keys --from-seed=$HOSTNAME)
-           cat <<< $($JQ --arg PEER $PEER '. += [$PEER]' $PEER_LIST_PATH) > $PEER_LIST_PATH
+
+           # Acquire lock and add $PEER to ${PEER_LIST_PATH} only once
+           (
+               flock --exclusive -w 10 201 || exit 1
```

Comment:
Great ! I was wondering if the `fd` was shared across container with shared volume, but I think it is.
One way to test the lock would be to add a `sleep` of 15 sec for one of the container just after the lock is acquired. That way, we should see the failure on other nodes.